README.md

```diff
@@ -216,6 +216,28 @@ module.exports = {
 }
 ```
 
+#### settings.kit
+
+If you use SvelteKit with not default configuration, you need to set below configurations.
+The schema is subset of SvelteKit's configuration.
+Therefore please check [SvelteKit docs](https://kit.svelte.dev/docs/configuration) for more details.
+
+e.g.
+
+```js
+module.exports = {
+  // ...
+  settings: {
+    kit: {
+      files: {
+        routes: "src/routes",
+      },
+    },
+  },
+  // ...
+}
```

Comment:
I choose that I added shared configuration for kit routes path.
I think over 95% of users use default settings `src/routes`.
So only a few users need to config it.

And for 5% users, we discussed that we will use `espree` but it may fail to get the path if they rely to env of some external information. So for now I didn't implement this logic.
https://github.com/ota-meshi/eslint-plugin-svelte/issues/241#issuecomment-1236031115